### ISSUE TITLE
Add section to address importing a poorly pool

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-doc#275]
Thanks to forum member musugru for helping to highlight this documentation short-fall. In some cases, such as disaster recovery, it is useful to be able to import poorly pools, such as those that will only mount ro, into a fresh install. Currently as of Rockstor 4.0.7 'Built on openSUSE' there is not custom mount option capability for the initial import.

The current work-around for this scenario, on a fresh install, is to first create the mount point:
```
mkdir /mnt2/<pool-label-here>
```
and then to mount the pool via the command line with the required options, i.e. for a poorly pool that requires a read-only mount and to have a pending balance not auto-restart, which is the default, one could use:
```
mount -o ro,skip_balance /dev/disk/by-id/<a-pool-member-disk-name> /mnt2/<pool-label-here>
```
The regular import mechanism within Rockstor via the disks page should then be able to import the named pool and establish the Web-UI required database entries and potentially facilitate back-up refresh etc. And hopefully aid in establishing the problem.

Note that a degraded option should also be mentioned for importing pools on a new install where that pool is know to be missing a member. The Web-UI can then facilitate guidance re repair/re-raid of that pool. But only once it has successfully imported the pool in the first place.

See the following forum post for context: https://forum.rockstor.com/t/reboot-although-still-in-pool-balancing-error-running-a-command-cmd/7825/5

**Comment by phillxnet** [comment]
Given we are now encountering many new installs that are importing prior potentially poorly pools this may well be a nice-to-have in the v4 stable milestone. Especially given 'poorly' is a far more wide ranging term, to our many years newer kernels in v4, than it was to our outdated 4.10/4.12 kernels in our prior v3 CentOS based offerings.

**Comment by phillxnet** [comment]
I am currently working on this issue.
